.changeset/olive-crabs-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Test and document small Button size variation
```

Comment:
As there's no material change to the package in this PR, let's remove this Changeset. It's mostly documentation and test updates.

Reply:
Reverted in c5445ad

Happy to remove both b643b34 and c5445ad via a rebase if you'd prefer a clean history in the branch. Happy to leave it too as you mentioned we squash into main